plugins/org.eclipse.sirius/src/org/eclipse/sirius/business/internal/session/danalysis/DAnalysisSessionImpl.java

```diff
@@ -1360,9 +1368,11 @@ public void close(IProgressMonitor monitor) {
             broker = null;
         }
 
-        CommandStackUtil.flushOperations(transactionalEditingDomain.getCommandStack());
-        // Unload all referenced resources
-        unloadAllResources(monitor);
+        if (transactionalEditingDomain != null) {
```

Comment:
Why not use `ted` defined above?

Reply:
Discussed with @mPorhel: this is on purpose to minimize the amount of code changed this close to the 7.4.0 release.

Reply:
I confirm, we should rationalize the method and get the ted in the same way every way, but so close to the release, I tried to minimize the changes